cleanlab/multilabel_classification/filter.py

```diff
@@ -134,10 +134,8 @@ def find_label_issues(
             warnings.warn(f"`rank_by_kwargs` is not used when `low_memory=True`.")
         arg_values = {
             "filter_by": filter_by,
-            "frac_noise": frac_noise,
             "num_to_remove_per_class": num_to_remove_per_class,
             "confident_joint": confident_joint,
-            "min_examples_per_class": min_examples_per_class,
```

Comment:
I think we should still check if users pass in non-default args for this (see comment on `frac_noise` above)